block_element_test.go

```diff
@@ -142,6 +142,20 @@ func TestNewPlainTextInputBlockElement(t *testing.T) {
 
 }
 
+func TestNewEmailTextInputBlockElement(t *testing.T) {
+	emailTextInputElement := NewEmailTextInputBlockElement(nil, "test@test.com")
```

Comment:
[IMO]
Prefer to use `example.com` for testing data.
(Existing test data should be corrected as well...)

Reply:
Done